calico/felix/config.py

```diff
@@ -441,7 +441,7 @@ def _validate_cfg(self, final=True):
             if parameter.value is None:
                 # No value, not even a default
                 raise ConfigException("Missing undefaulted value",
-                                      self.parameters["InterfacePrefix"])
+                                      name)
```

Comment:
Doesnt' the opbject in self.parameters stringify to a nice description of the param?  I.e. maybe the line was correct?